antenna/breakpad_resource.py

```diff
@@ -206,15 +204,22 @@ def extract_payload(self, req):
                         # The UnicodeDecodeError can happen if the utf-8 codec can't decode
                         # one of the characters. The JSONDecodeError can happen in a variety
                         # of "malformed JSON" situations.
-                        raise MalformedCrashReport("bad_json")
+                        raise MalformedCrashReport("invalid_json")
 
                     if not isinstance(raw_crash, dict):
-                        raise MalformedCrashReport("bad_json")
+                        raise MalformedCrashReport("invalid_json_value")
 
                 elif part.content_type.startswith("text/plain"):
```

Comment:
I should add a check to see if there's a filename. If there's a filename, then this can't be a text annotation.